app/models/concerns/hyrax/ability.rb

```diff
@@ -113,7 +113,7 @@ def proxy_deposit_abilities
           end
         end
 
-        can :create, ProxyDepositRequest if Flipflop.proxy_deposit? && registered_user?
+        can :create, ProxyDepositRequest if (Flipflop.proxy_deposit? || Flipflop.transfer_works?) && registered_user?
```

Comment:
@blancoj Why do we have a dependency on `proxy_deposit` for a user to be able to transfer ownership of a work? As I understand, only enabling `transfer_works` should give the user the ability to transfer ownership of their work. I have so little knowledge on these, so why do we have a check for both these?

Reply:
@Dananji  The `ProxyDepositRequest` model seems to be shared by both proxy and transfer functionality.  My guess is that at one time they were inter dependent.  https://github.com/samvera/hyrax/blob/3af966faf937f10446e4983ca1f384df5ca9c2da/app/models/proxy_deposit_request.rb#L1